diff --git a/node_modules/mocha-chai-jest-snapshot/dist/utils/jest-test-result-helper.d.ts b/node_modules/mocha-chai-jest-snapshot/dist/utils/jest-test-result-helper.d.ts
index 7ce132e..30d41a0 100644
--- a/node_modules/mocha-chai-jest-snapshot/dist/utils/jest-test-result-helper.d.ts
+++ b/node_modules/mocha-chai-jest-snapshot/dist/utils/jest-test-result-helper.d.ts
@@ -7,6 +7,7 @@
 import { SnapshotSummary, TestResult } from "@jest/test-result";
 import { SnapshotState } from "jest-snapshot";
 declare type SnapshotResult = TestResult["snapshot"];
+declare type SnapshotStateOptions = ConstructorParameters<typeof SnapshotState>[1];
 export { SnapshotSummary };
 export declare const makeEmptySnapshotSummary: (options: SnapshotStateOptions) => SnapshotSummary;
 export declare const packSnapshotState: (snapshotState: SnapshotState) => SnapshotResult;
