diff --git a/node_modules/sparqljs/lib/SparqlGenerator.js b/node_modules/sparqljs/lib/SparqlGenerator.js
index 527f0e9..ca03f98 100644
--- a/node_modules/sparqljs/lib/SparqlGenerator.js
+++ b/node_modules/sparqljs/lib/SparqlGenerator.js
@@ -110,6 +110,10 @@ Generator.prototype.bgp = function (bgp) {
   return this.encodeTriples(bgp.triples);
 };
 
+Generator.prototype.comment = function(comment) {
+  return comment.text.split('\n').map((line) => `# ${line}`).join(this._newline);
+}
+
 Generator.prototype.encodeTriples = function (triples) {
   if (!triples.length)
     return '';
